src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Buttons/BitToggleButtonTest.razor

```diff
@@ -0,0 +1,37 @@
+﻿<CascadingValue Value="Visual">
+    <BitToggleButton @bind-IsChecked="@IsChecked"
+                     ChekedLabel="@ChekedLabel"
+                     UnChekedLabel="@UnChekedLabel"
+                     CheckedIconName="@CheckedIconName"
+                     UnCheckedIconName="@UnCheckedIconName"
+                     AllowDisabledFocus="AllowDisabledFocus"
+                     IsEnabled="IsEnabled"
+                     ButtonStyle="ButtonStyle"
+                     AriaDescription="@AriaDescription"
+                     AriaLabel="@AriaLabel"
+                     AriaHidden="AriaHidden">
+    </BitToggleButton>
+</CascadingValue>
+
+@code {
+    [Parameter] public Visual Visual { get; set; }
+    [Parameter] public bool AllowDisabledFocus { get; set; }
+    [Parameter] public string? AriaDescription { get; set; }
+    [Parameter] public bool AriaHidden { get; set; }
+    [Parameter] public string? AriaLabel { get; set; }
+    [Parameter] public bool IsEnabled { get; set; }
+    [Parameter] public bool IsChecked { get; set; }
+    [Parameter] public string? CheckedIconName { get; set; }
+    [Parameter] public string? UnCheckedIconName { get; set; }
+    [Parameter] public string? ChekedLabel { get; set; }
+    [Parameter] public string? UnChekedLabel { get; set; }
+    [Parameter] public ButtonStyle ButtonStyle { get; set; }
+
+
+    public int CurrentCount { get; set; }
+
+    public void IncrementCount(MouseEventArgs e)
```

Comment:
where has this method been used? I think it should be used as the OnClick event handler of the BitToggleButton component